internal/statestore/backfill.go

```diff
@@ -134,3 +139,70 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 
 	return nil
 }
+
+// AcknowledgeBackfill - store Backfill's last acknowledgement time
+func (rb *redisBackend) AcknowledgeBackfill(ctx context.Context, id string) error {
+	redisConn, err := rb.redisPool.GetContext(ctx)
+	if err != nil {
+		return status.Errorf(codes.Unavailable, "AcknowledgeBackfill, id: %s, failed to connect to redis: %v", id, err)
+	}
+	defer handleConnectionClose(&redisConn)
+
+	currentTime := time.Now().UnixNano()
```

Comment:
@Laremere is there an agreement of not using UTC()? `time.Now()` is used everywhere.

Reply:
Assumption is all computers in a cluster will have clocks that are synchronized enough.